package.json

```diff
@@ -29,7 +29,8 @@
     "test:all": "ember try:each"
   },
   "dependencies": {
-    "broccoli-funnel": "^2.0.1",
+    "@ember/optional-features": "^0.6.3",
```

Comment:
why did you move this to the `dependencies` section?

Reply:
@turbo87 the optional feature APIs cannot be accessed at build time without it being in dependencies, it seems.

Reply:
I guess you could assume that if the addon is not available then jQuery wasn't switched off 🤔

Reply:
Yeah, I think I tried that and it wasn't picking up dependencies from the host app in `this.addons`, but maybe I needed to do something fancy, like traverse my way up to the app or something. I'll dig through the Ember-CLI innards again and see if I can figure out a way to remove the dependency and use the host's, if it's available.